batch/batch/client.py

```diff
@@ -111,11 +99,29 @@ def delete(self):
 
 
 class BatchClient:
-    def __init__(self, url=None, timeout=None, cookies=None, headers=None):
+    HAIL_NOTEBOOK_JWT_TOKEN_LOCATION = '/user-jwt/jwt'
+
+    def __init__(self, url=None, timeout=None, token_file=None, token=None, headers=None):
+        if token_file is not None and token is not None:
+            raise ValueError('set only one of token_file and token')
         if not url:
             url = 'http://batch.default'
         self.url = url
-        self.api = api.API(timeout=timeout, cookies=cookies, headers=headers)
+        if token is None:
+            token_file = (token_file or
+                          os.environ.get('HAIL_TOKEN_FILE') or
+                          os.path.expanduser('~/.hail/token'))
+            if not os.path.exists(token_file):
```

Comment:
this feels like it should be part of the above if statement

```python
token_file = (token_file or
                           os.environ.get('HAIL_TOKEN_FILE') or
                           os.path.expanduser('~/.hail/token') or
                           '/user-jwt/jwt'
```

Reply:
oops forgot to remove from client.py too, fixed